probes/http/http.go

```diff
@@ -129,10 +131,23 @@ func (p *Probe) Init(name string, opts *options.Options) error {
 
 	// Needs to be non-nil so we can set parameters on it.
 	transport := http.DefaultTransport
-
 	// Keep idle connections open until we explicitly close them.
 	// This allows us to send multiple requests over the same connection.
 	transport.(*http.Transport).MaxIdleConnsPerHost = 1
+	if p.protocol == "https" {
+		tlsConfig := &tls.Config{}
+		tlsConfig.InsecureSkipVerify = p.c.GetAllowInsecureTls()
+		if p.c.TlsCaPath != nil {
+			caCert, err := ioutil.ReadFile(p.c.GetTlsCaPath())
+			if err != nil {
+				p.l.Errorf("Invalide Path for Ca Certs %s", p.c.GetTlsCaPath())
```

Comment:
We should not continue after this error. If you do this check in initialization, you can simply return error.